tasks/actorstate/task.go

```diff
@@ -41,6 +43,25 @@ func NewTaskWithTransformer(node tasks.DataSource, extractorMap ActorExtractorMa
 	}
 }
 
+const actorChangeLimiterEnv = "LILY_ACTOR_CHANGE_LIMITER"
+
+const defaultActorChangeLimit = 10000
+
+// Skip task processing when actor state changes abnormally.
+func (t *Task) exceedActorChangeLimit(changes map[address.Address]tasks.ActorStateChange) bool {
+	// The 10,000 is a default threshold
+	limit := defaultActorChangeLimit
+
+	// Access the custom threshold set in the environment variable
+	s := os.Getenv(actorChangeLimiterEnv)
```

Comment:
```suggestion
	s := os.Getenv(actorChangeLimiterEnv)
```

[os.LookupEnv](https://pkg.go.dev/os#LookupEnv) might be nicer.

Reply:
nice